gnomad_qc/v2/resources/basics.py

```diff
@@ -409,19 +409,19 @@ def genomes_exomes_duplicate_ids_tsv_path(version: str = CURRENT_DUPS) -> str:
 
 
 def omni_mt_path(hail_version=CURRENT_HAIL_VERSION):
-    return 'gs://gnomad-public/truth-sets/hail-{0}/1000G_omni2.5.b37.mt'.format(hail_version)
+    return 'gs://gcp-public-data--gnomad/truth-sets/hail-{0}/1000G_omni2.5.b37.mt'.format(hail_version)
 
 
 def mills_mt_path(hail_version=CURRENT_HAIL_VERSION):
-    return 'gs://gnomad-public/truth-sets/hail-{0}/Mills_and_1000G_gold_standard.indels.b37.mt'.format(hail_version)
+    return 'gs://gcp-public-data--gnomad/truth-sets/hail-{0}/Mills_and_1000G_gold_standard.indels.b37.mt'.format(hail_version)
 
 
 def hapmap_mt_path(hail_version=CURRENT_HAIL_VERSION):
-    return 'gs://gnomad-public/truth-sets/hail-{0}/hapmap_3.3.b37.mt'.format(hail_version)
+    return 'gs://gcp-public-data--gnomad/truth-sets/hail-{0}/hapmap_3.3.b37.mt'.format(hail_version)
 
 
 def kgp_high_conf_snvs_mt_path(hail_version=CURRENT_HAIL_VERSION):
-    return 'gs://gnomad-public/truth-sets/hail-{0}/1000G_phase1.snps.high_confidence.b37.mt'.format(hail_version)
+    return 'gs://gcp-public-data--gnomad/truth-sets/hail-{0}/1000G_phase1.snps.high_confidence.b37.mt'.format(hail_version)
 
 
 def kgp_phase3_genotypes_mt_path(split: bool = True, hail_version=CURRENT_HAIL_VERSION) -> str:
```

Comment:
Doesn't actually matter since the correct resource will be used, but the function description still has `gs://gnomad-public` instead of `gs://gcp-public-data--gnomad/truth-sets/hail-0.2/1000G.GRCh38.20130502.phase3.sequence.index`

Reply:
Will fix -- good catch!